findme-location-riddles/src/LocationRiddlesRepository.py

```diff
@@ -57,11 +44,12 @@ def get_all_location_riddles_by_user_id(self, user_id):
         try:
```

Comment:
just FYI no change needed: boto3 paginators solve the problem above (L36...) nicely https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html